CHANGELOG.next.asciidoc

```diff
@@ -352,6 +352,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update filebeat httpjson input to support pagination via Header and Okta module. {pull}16354[16354]
 - Added new Checkpoint Syslog filebeat module. {pull}17682[17682]
 - Add config option to select a different azure cloud env in the azure-eventhub input and azure module. {issue}17649[17649] {pull}17659[17659]
+- Improve ECS categorization field mappings for nats module. {issue}16173[16173] {pull}17550[17550]
```

Comment:
This line should not be added to the changelog.

Reply:
Thanks! I have just already fixed it